sklearn/metrics/classification.py

```diff
@@ -1576,6 +1577,11 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    .. versionadded:: 0.18
```

Comment:
`versionadded` goes below the new thing, indented on the same level as `If not provided`.